spring-cloud-build/pom.xml

```diff
@@ -125,6 +128,13 @@ limitations under the License.
 					<artifactId>asciidoctor-maven-plugin</artifactId>
 					<version>1.5.2</version>
 				</plugin>
+				<plugin>
+					<groupId>org.apache.maven.plugins</groupId>
+					<artifactId>maven-surefire-plugin</artifactId>
+					<configuration>
+						<redirectTestOutputToFile>true</redirectTestOutputToFile>
```

Comment:
We removed `-Dmaven.test.redirectTestOutputToFile=true` where it was problematic.  I just want to be able to see test output.  I don't care how.